great_expectations/datasource/data_connector/util.py

```diff
@@ -17,41 +17,19 @@
 from great_expectations.data_context.util import instantiate_class_from_config
 from great_expectations.datasource.data_connector.asset import Asset  # noqa: TCH001
 from great_expectations.datasource.data_connector.sorter import Sorter  # noqa: TCH001
+from great_expectations.optional_imports import (
+    BlobPrefix,
+    BlobServiceClient,
+    ContainerClient,
+    google_cloud_storage,  # noqa: F401
+)
 
 if TYPE_CHECKING:
     from great_expectations.alias_types import PathStr
     from great_expectations.datasource import DataConnector
 
 logger = logging.getLogger(__name__)
 
-try:
-    from azure.storage.blob import BlobPrefix, BlobServiceClient, ContainerClient
-except ImportError:
-    BlobPrefix = None
-    BlobServiceClient = None
-    ContainerClient = None
-    logger.debug(
-        "Unable to load azure types; install optional Azure dependency for support."
-    )
-
-try:
-    from google.cloud import storage
-except ImportError:
-    storage = None
-    logger.debug(
-        "Unable to load GCS connection object; install optional Google dependency for support"
-    )
-
-try:
-    import pyspark
```

Comment:
:heart: I think our new pattern might help us clean up unused imports like this. That's awesome.